bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,942 @@
+				<description>Default is 230 UDP.
+				</description>
+				<default>230</default>
+			</parameter>
+			<parameter name="PREFERRED_LOCAL_PORT_NUMBER" type="integer" required="true">
+				<label>Preferred local port number</label>
+				<description>Default port is 0. It means that it is random.
+				</description>
+				<default>0</default>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="PING_INTERVAL" type="integer" min="1" max="300">
+				<label>Ping interval</label>
+				<description>Interval in seconds to check for device presence.</description>
+				<default>30</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="SUBSCRIBTION_INTERVAL" type="integer" min="1" max="60">
+				<label>Subscription interval</label>
+				<description>Interval in minutes to subcribe Souliss Gateway.
+				</description>
+				<default>2</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="HEALTHY_INTERVAL" type="integer" min="1" max="60">
+				<label>Healthy interval</label>
+				<description>Interval in seconds to send nodes healthy.
+				</description>
+				<default>35</default>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="SEND_INTERVAL" type="integer" min="5" max="5000">
+				<label>Send interval</label>
+				<description>Interval in milliseconds to get packet from binding queue and send it to Souliss. First packet is sent
+					immediately.
+				</description>
+				<default>1000</default>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="TIMEOUT_TO_REQUEUE" type="integer" min="0" max="10000">
+				<label>Interval in milliseconds to requeue packet in queue if not yet executed</label>
+				<description></description>
+				<required>true</required>
+				<default>5000</default>
+			</parameter>
+
+			<parameter name="TIMEOUT_TO_REMOVE_PACKET" type="integer" min="0" max="60000">
+				<label>Interval in milliseconds to remove packet from queue if not yet executed </label>
+				<description></description>
+				<required>true</required>
+				<default>20000</default>
+			</parameter>
+
+			<parameter name="USER_INDEX" type="integer" required="true">
+				<label>User Index</label>
+				<description>Generally the default value work good. It must is different from other user interfaces (ex: SoulissApp)
+				</description>
+				<default>70</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="NODE_INDEX" type="integer" required="true">
+				<label>Node Index</label>
+				<description>Generally the default value work good. It must is different from other user interfaces (ex: SoulissApp)
+				</description>
+				<default>120</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="t11">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+		<label>T11</label>
+		<description>Simple Light</description>
```

Comment:
T11 seems a little strange to have as a label, should it be `Simple Light` here? Since I dont know the protocol details I do not know about the how it is best used or if this makes sense when better informed?